database.py

```diff
@@ -1561,7 +1561,7 @@ def field_name(self):
 
         def startswith_counter(prefix, names):
             """ Return the number of strings in 'names' starting with 'prefix' """
-            return len(x for x in names if x.startswith(prefix))
```

Comment:
Thanks for fixing this. For future reference, this results in error "object of type 'generator' has no len()" I'm surprised this went unnoticed for so many years.